spec/features/publish/editing_visa_sponsorships_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
```

Comment:
This file should probably go in `features/publish_interface` pending any further decisions about what this namespace is called.